nifi-nar-bundles/nifi-standard-services/nifi-hbase_1_1_2-client-service-bundle/nifi-hbase_1_1_2-client-service/src/main/java/org/apache/nifi/hbase/HBase_1_1_2_ClientService.java

```diff
@@ -359,43 +392,45 @@ protected Connection createConnection(final ConfigurationContext context) throws
         }
 
         if (SecurityUtil.isSecurityEnabled(hbaseConfig)) {
-            String principal = context.getProperty(kerberosProperties.getKerberosPrincipal()).evaluateAttributeExpressions().getValue();
-            String keyTab = context.getProperty(kerberosProperties.getKerberosKeytab()).evaluateAttributeExpressions().getValue();
-            String password = context.getProperty(kerberosProperties.getKerberosPassword()).getValue();
-
-            // If the Kerberos Credentials Service is specified, we need to use its configuration, not the explicit properties for principal/keytab.
-            // The customValidate method ensures that only one can be set, so we know that the principal & keytab above are null.
-            final KerberosCredentialsService credentialsService = context.getProperty(KERBEROS_CREDENTIALS_SERVICE).asControllerService(KerberosCredentialsService.class);
-            if (credentialsService != null) {
-                principal = credentialsService.getPrincipal();
-                keyTab = credentialsService.getKeytab();
-            }
+            getLogger().info("HBase Security Enabled, creating KerberosUser");
```

Comment:
Is this necessary as an info log? Perhaps changing this to a debug log is sufficient given the info log message on successful login.